modules/AOW_Actions/actions/actionCreateRecord.js

```diff
@@ -44,7 +44,7 @@ function show_crModuleFields(ln){
 
                 //document.getElementById('aow_action_div'+ln).innerHTML ="<select tabindex='116' name='service_vat[]' id='service_vat'>" + cr_fields[ln] + "</select>";
             }
-        }
+        };;;;;;;
```

Comment:
LOL. I just really wanted to make sure that the statement had ended. :D

I have removed it.